apps/src/music/progress/MusicValidator.ts

```diff
@@ -125,7 +125,7 @@ export default class MusicValidator extends Validator {
               value: eventData.functionContext.name,
             });
             this.conditionsChecker.addSatisfiedCondition({
-              name: MusicConditions.PLAYED_SOUND_IN_FUNCTION_BY_ID.name,
+              name: MusicConditions.PLAYED_SOUND_IN_FUNCTION.name,
               value: eventData.functionContext.functionCallBlockId,
```

Comment:
Should we rename `functionCallBlockId` everywhere now? It could `procedureModelId` or even just `procedureId`.

Reply:
Good call.  Updated.